.vscode/tasks.json

```diff
@@ -14,44 +14,8 @@
         "focus": false,
         "panel": "dedicated"
       },
-      "args": ["run", "prepack"],
-      "isBackground": false,
-      "problemMatcher": {
-        "owner": "typescript",
-        "fileLocation": "relative",
-        "pattern": {
-          "regexp": "^(.*\\.ts):(\\d*):(\\d*)(\\s*-\\s*)(error|warning|info)\\s*(TS\\d*):\\s*(.*)$",
-          "file": 1,
-          "line": 2,
-          "column": 3,
-          "severity": 5,
-          "code": 6,
-          "message": 7
-        }
-      }
-    },
-    {
-      "label": "Lint",
-      "command": "yarn",
-      "type": "shell",
-      "presentation": {
-        "focus": false,
-        "panel": "dedicated"
-      },
-      "args": ["run", "lint"],
-      "isBackground": false,
-      "problemMatcher": {
-        "owner": "typescript",
-        "fileLocation": "relative",
-        "pattern": {
-          "regexp": "^(ERROR|WARNING|INFO):\\s*(.*\\.ts):(\\d*):(\\d*)(\\s*-\\s*)(.*)$",
-          "file": 2,
-          "line": 3,
-          "column": 4,
-          "severity": 1,
-          "message": 6
-        }
-      }
+      "args": ["run", "pretest"],
```

Comment:
runs `yarn run pretest` before launching the node process, the `pretest` script compiles the tests, if there's a TS error in a test vscode will show a warning and halt the process.